pkgs/top-level/all-packages.nix

```diff
@@ -11893,14 +11893,19 @@ in
 
   ao = callPackage ../applications/graphics/ao {};
 
-  ardour = ardour4;
+  ardour = ardour5;
 
   ardour3 =  callPackage ../applications/audio/ardour/ardour3.nix {
     inherit (gnome2) libgnomecanvas libgnomecanvasmm;
     inherit (vamp) vampSDK;
   };
 
-  ardour4 =  callPackage ../applications/audio/ardour {
+  ardour4 =  callPackage ../applications/audio/ardour/ardour4.nix {
```

Comment:
I left it in because it seems like there are still some midi related bugs in the new release. 
Also with each new release the binary name gets a suffix with the major version number
(e.g. for 4.7 its `ardour4`, and for 5.4 now `ardour5`. So it seems like it would make sense that
both versions are available, which is also like  [they are handled for fedora](https://admin.fedoraproject.org/pkgdb/packages/ardour%2A/)